test/tests/providers/global/inhibit.prometheus-agent.rules.test.yml

```diff
@@ -0,0 +1,32 @@
+---
+rule_files:
```

Comment:
Can you add a test for the other inhibition

Reply:
2 test cases for both inhibition.